src/packages/cli/package.json

```diff
@@ -113,7 +114,8 @@
     "terminal-link": "2.1.1",
     "ts-jest": "26.1.4",
     "ts-mocha": "7.0.0",
-    "typescript": "3.9.7"
+    "typescript": "3.9.7",
+    "ws": "^7.3.1"
```

Comment:
I tried to not add dependencies as much as possible, but I needed this to be able to communicate with Studio server, since a vanilla Node implementation would be extremely verbose.

I also thought about exposing the Websocket endpoint in studio-server as a REST endpoint, but that turned out to be much more complicated too.